### PR TITLE
Add a method to select all items

### LINKED ISSUE #995: Add an option to select all items
**Thread**
https://forum.gonevis.com/d/28-option-to-select-all-posts-to-do-actions

### PR DESCRIPTION
**Pull request by ArsalanSavand**
Fixes #995 

- Screenshots

image

### CODE REVIEW

File: src/app/basement/multi_select/multi_select_view.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!-- Action dropdown -->
+<div class="pull-left select-all">
+  <input type="checkbox" ng-model="$ctrl.isAllSelected" ng-change="$ctrl.selectAll()">
+  <span ng-if="$ctrl.countSelected > 0">{{ $ctrl.countSelected }} selected</span>
```
Comment (Alir3z4) [inline src/app/basement/multi_select/multi_select_view.html]: Mark for translation.

File: src/app/entry/entry_controller.js [hunk 2]
```diff
@@ -28,37 +28,46 @@ function EntryController($scope, $state, $stateParams, Entry, UndoService, Codek
...
+        }]
       }, {
-        label: translation.PUBLISHED,
-        icon: "globe",
-        property: "status",
-        value: 1,
-      }, {
-        label: translation.PIN,
-        icon: "star",
+        label: "Set feature",
```
Comment (Alir3z4) [inline src/app/entry/entry_controller.js]: **Set Featured**

Also mark for translations.

File: src/app/entry/entry_controller.js [hunk 3]
```diff
@@ -28,37 +28,46 @@ function EntryController($scope, $state, $stateParams, Entry, UndoService, Codek
...
+        description: "Pin to front page",
```
Comment (Alir3z4) [inline src/app/entry/entry_controller.js]: Also mark for translations.

File: src/app/entry/entry_controller.js [hunk 4]
```diff
@@ -28,37 +28,46 @@ function EntryController($scope, $state, $stateParams, Entry, UndoService, Codek
...
+        }]
       }, {
-        label: translation.UNPIN,
-        icon: "star-o",
-        property: "featured",
-        value: false
-      }, {
-        label: translation.ENABLE_COMMENTS,
-        icon: "comments",
+        label: "Commenting",
```
Comment (Alir3z4) [inline src/app/entry/entry_controller.js]: Mark for translations.

File: src/app/entry/entry_controller.js [hunk 5]
```diff
@@ -28,37 +28,46 @@ function EntryController($scope, $state, $stateParams, Entry, UndoService, Codek
...
+        description: "Change commenting",
```
Comment (Alir3z4) [inline src/app/entry/entry_controller.js]: Mark for translations.

File: src/style/css/main.css [hunk 6]
```diff
@@ -1739,6 +1739,62 @@ img.cache {
 
 /* Entry */
 
+.post-list {
```
Comment (Alir3z4) [inline src/style/css/main.css]: Can't we have all of these post lists CSS stuff in their module to do lazy loading ?

**Comment by Alir3z4** [comment]
@ArsalanSavand Please address my comments.
@AmirSavand We need your review on this as well.

File: src/app/basement/multi_select/multi_select_view.html (hunk 1 above)
Comment (ArsalanSavand) [reply src/app/basement/multi_select/multi_select_view.html]: Done.

File: CHANGELOG.md [hunk 7]
```diff
@@ -5,6 +5,8 @@
 - Resolved #993: Rename "Files" to "Media".
 - Fixed #991: Fix pressing CTRL + Z (undo) will fully clear editor's content.
 - Resolved #998: Rename "Files" to "Images" at image selection modal.
+- Resolved #998: Rename "Files" to "Images" at image selection modal.
```
Comment (AmirSavand) [inline CHANGELOG.md]: Remove this.

**Pull request merged**